pkg/utils/ids_migration_helper.go

```diff
@@ -0,0 +1,62 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"strings"
+)
+
+var Host string
```

Comment:
I'd recommend instead storing `var Region string` here, and when you set the parameter during initialization you can extract the region once then, rather than extracting it from the hostname every time `TransfromIdFromRegion` is called. Also then you can error during initialization instead of in `TransformIdWithRegion`, so you won't need to return an error there.